resources/views/autor.blade.php

```diff
@@ -3,7 +3,7 @@
 @section('og')
 <meta property="og:title" content="{!! $author->formatedName !!}" />
 
-<meta property="og:description" content="{!! $author->getDescription(false, false, true) !!}" />
+<meta property="og:description" content="{{ $author->getDescription() }}" />
```

Comment:
As I mentioned above, it might be worth trying to encapsulate the description in a component, and then do something like

```suggestion
<meta property="og:description" content="{{ html_strip(@component('components.author_description', $author) }}" />
```
I honestly dont know if it'll work but if it does, we could delete some more code :)